### ISSUE TITLE
Move this repo to the LSE-Methodology organization

### ISSUE THREAD

**Issue by kbenoit** [LSE-Methodology/MY451#2]
_From @kbenoit on September 20, 2017 13:15_

@amatsuo could you move this to the LSE-Methodology organization? This will involve some YAML changes and maybe a few other configuration changes. 

Since we started this, it's become possible to publish from a designated sub directory, making the use of the `gh-pages` branch unnecessary. Might be worth using that structure.

Thanks!

_Copied from original issue: kbenoit/coursepack-bookdown#21_

**Comment by kbenoit** [comment]
_From @amatsuo on September 20, 2017 13:26_

Isn't it just doing this? 

https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/

**Comment by kbenoit** [comment]
Normally yes but for this transfer there are also yaml config files and possibly some hard coded URL references.

Plus there is the move from gh-pages part.

**Comment by kbenoit** [comment]
_From @amatsuo on September 21, 2017 10:46_

The website is up with old contents. 
https://lse-methodology.github.io/MY451/

I also updated the travis settings on several .yml files. This would already work once a travis hook is set up. Could you make Travis watch the repo "LSE-Methodology/MY451"?

**Comment by kbenoit** [comment]
I can provide permissions for you to set this up - what do you need?

**Comment by kbenoit** [comment]
_From @amatsuo on September 21, 2017 11:21_

I need your travis login, if you don't mind.

**Comment by kbenoit** [comment]
_From @chmue on September 26, 2017 16:42_

What is the status of the transfer? Is it finished or are there still things to do?

I'm asking because I pointed Jon (@jacksojp) to the new repo so there are new changes in the fork.

**Comment by kbenoit** [comment]
@amatsuo moved it and it appeared to be ok. Note that no local processing is required - it should work by simply pushing a markdown file and the CI will build the website.

**Comment by kbenoit** [comment]
_From @chmue on September 26, 2017 18:12_

Ok, great! Are there any plans to incorporate the changes from the fork here, then delete the fork and transfer this to LSE-Methodology?

The reason I'm asking is that this is still the main repo and the one at the LSE-Methodology is just a fork. That means you cannot do certain things on the new (forked) repo such as reporting issues and text search of the content.

**Comment by kbenoit** [comment]
Yes that’s entirely the idea, to get it out of my user account and move it entirely to the LSE-Methodology organization. If it’s working in LSE-Methodology, we can just delete it from my account. (I assume this will not kill the “downstream” fork, if that even makes sense in git-speak.)

**Comment by kbenoit** [comment]
_From @chmue on September 26, 2017 18:17_

Hm, I'm actually not sure off the top of my head if there are any issues with just deleting upstream. The Git history itself should be safe in the fork but I'm not sure if GH does some special things to upstream/fork that would break.

**Comment by kbenoit** [comment]
> ### Deleting a private repository
>
> When you delete a private repository, all of its private forks are also deleted

From https://help.github.com/articles/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility/

Sounds like we should make it public first. I’ll do that now.

**Comment by kbenoit** [comment]
_From @chmue on September 26, 2017 18:27_

Maybe it's easier to just merge the changes in the fork and then transfer ownership?

Edit: Ah no, sorry! I should have read until the very end of the page. You want to make the repo public and then delete it right after that such that the private fork gets its own history tree.

**Comment by kbenoit** [comment]
_From @chmue on September 26, 2017 18:40_

I think it worked! The other repo is not a fork anymore. 

The issues seem to be missing but I am not sure if you have to add an issue tracker separately. We might need to copy over the issues from here by hand. I'm happy to look into this if you give me full access to the other repo.

**Comment by kbenoit** [comment]
Done.

**Comment by kbenoit** [comment]
_From @chmue on September 27, 2017 11:27_

Could you also make me an owner of this repo, @kbenoit ? I am using [this tool](https://github-issue-mover.appspot.com/) to move the issues. It tells me that I have the required access to LSE-Methodology/MY451 but not for this repo.

**Comment by kbenoit** [comment]
You have admin rights, but LSE-Methodology is the "owner". If I transfer ownership then it moves to your account.

<img width="937" alt="screenshot 2017-09-27 13 56 03" src="https://user-images.githubusercontent.com/2182246/30914495-c62e8cac-a38b-11e7-8910-274e3e36a499.png">

**Comment by kbenoit** [comment]
_From @chmue on September 27, 2017 13:1_

Ok. I *do* have the right access for `LSE-Methodology/MY451`. But I don't have the right access to this repo `kbenoit/coursepack-bookdown`.

**Comment by kbenoit** [comment]
See Slack on the brexit team

**Comment by chmue** [comment]
The move should be complete now.

I was only able to copy over the open issues so we lose the closed ones but I think that should be ok. If that is ok, you should be able to delete the old `kbenoit/coursepack-bookdown` repo now.
